src/sample/Controller.java

```diff
@@ -1,4 +1,59 @@
 package sample;
 
-public class Controller {
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyEvent;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable {
+    ObservableList<Contact> items = FXCollections.observableArrayList();
+
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField name;
+
+    @FXML
+    TextField phone;
+
+    @FXML
+    TextField email;
+
+
+    public void addItems() {
+        items.add(new Contact(name.getText(), phone.getText(), email.getText()));
```

Comment:
you need a line above line 32 that stops the user from adding a blank text field to the contacts.  Everything else looks good.  Add that line and resubmit.